opensrp-app/src/main/java/org/smartregister/repository/AllSharedPreferences.java

```diff
@@ -385,5 +387,24 @@ public int getDBEncryptionVersion() {
     public void setDBEncryptionVersion(int encryptionVersion) {
         preferences.edit().putInt(DB_ENCRYPTION_VERSION, encryptionVersion).commit();
     }
+
+    @Nullable
+    public String getUserPractitionerRole() {
+        return preferences.getString(USER_PRACTITIONER_ROLE, null);
+    }
+
+    @Nullable
+    public String getUserPractitionerIdentifier() {
+        return preferences.getString(USER_PRACTITIONER_IDENTIFIER, null);
+    }
+
+    public void setUserPractitionerRole(String practitionerRole) {
+        preferences.edit().putString(USER_PRACTITIONER_ROLE, practitionerRole).commit();
+    }
+
+    public void setUserPractitionerIdentifier(String identifier) {
+        preferences.edit().putString(USER_PRACTITIONER_IDENTIFIER, identifier).commit();
```

Comment:
OpenSRP supports multi tenancy on the same device where two user accounts can be active without requiring a data clearing or a fresh installation. Should then the preference keys here have the **+username** suffix like other user-dependent preference keys?